clients/src/main/java/org/apache/kafka/clients/telemetry/ClientTelemetryUtils.java

```diff
@@ -385,4 +403,45 @@ public static PushTelemetryRequest.Builder createPushTelemetryRequest(boolean te
             metricsData);
     }
 
+    public static ResourceLabel createConsumerLabels(final ConsumerGroupMetadata groupMetadata) {
+        if(groupMetadata == null) {
+            return new ResourceLabel();
+        }
+
+        ResourceLabel resourceLabels = new ResourceLabel();
```

Comment:
Do you know if the group metadata is available at the time of construction? Or is it something that has to be retrieved from the cluster on first contact?

Reply:
Actually - good point, I think it is available via the coordinator, which is initialized after the client telemetry. I'll look into it